test_lib/compaction.py

```diff
@@ -35,7 +37,10 @@ def get_compaction_strategy(node, keyspace, table):
             continue
         list_stripped_values = [val.strip() for val in row.split('|')]
         if list_stripped_values[0] == keyspace and list_stripped_values[1] == table:
-            compaction = CompactionStrategy.from_str(output_str=list_stripped_values[2].split("'")[3])
+            dict_compaction_values = yaml.load(list_stripped_values[2])
```

Comment:
can you clarify why yaml module is needed here ?

Reply:
yes, in order to convert a string to a dictionary. so it also validates a correct stored format and uses the needed key out of the generated dictionary.
example:
```
s
"{'bucket_high': '1.67', 'bucket_low': '0.79', 'class': 'IncrementalCompactionStrategy', 'max_threshold': '28', 'min_sstable_size': '68', 'min_threshold': '3', 'sstable_size_in_mb': '1699'}"
y = yaml.load(s)
y
{'bucket_high': '1.67', 'bucket_low': '0.79', 'class': 'IncrementalCompactionStrategy', 'max_threshold': '28', 'min_sstable_size': '68', 'min_threshold': '3', 'sstable_size_in_mb': '1699'}
type(y)
<class 'dict'>
y['class']
'IncrementalCompactionStrategy'
```

Reply:
`json.loads()` doesn't do the exact same thing ?